File: app/src/main/java/at/shockbytes/dante/util/books/BookListener.java
package at.shockbytes.dante.util.books;

/**
 * @author Martin Macheiner
 *         Date: 29.08.2016.
 */
public interface BookListener {

    void onBookAdded(Book book);

    void onBookDeleted(Book book);

    void onBookStateChanged(Book book, Book.State state);
}

File: app/src/main/java/at/shockbytes/dante/util/books/BookListener.kt
package at.shockbytes.dante.util.books

/**
 * @author Martin Macheiner
 * Date: 29.08.2016.
 */

interface BookListener {

    fun onBookAdded(book: Book)

    fun onBookDeleted(book: Book)

    fun onBookStateChanged(book: Book, state: Book.State)
}
